Month1/Week2/Section2/Month1-Week2-Section2/Spotify/Song.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+#include <string>
+#include <iostream>
+#include <string>
+class FSong
+{
+private:
+    int MinutesDuration;
+    int SecondsDuration;
+    std::string ArtistName;
+    std::string SongName;
+public:
+    FSong();
+
+    FSong(int EntryMinutesDuration, int EntrySecondsDuration, std::string EntryArtistName, std::string EntrySongName);
+    std::string GetFormatedDuration()const;
+    bool bIsvalid();
```

Comment:
Remember to use PascalCase and functions don't need to have a b prefix. The documentation has some examples for boolean functions.
![image](https://user-images.githubusercontent.com/36243539/181571456-e3ead0f1-d5d3-49c5-a6f4-de85879a6b4c.png)

Reply:
ok, I understand, changed